esphomeflasher/__main__.py

```diff
@@ -159,8 +159,8 @@ def run_esphomeflasher(argv):
     print("Done! Flashing is complete!")
     print()
 
-    if args.upload_baud_rate != 512000:
-        stub_chip._port.baudrate = 512000
+    if args.upload_baud_rate != 115200:
```

Comment:
Upload speed should kept the same. Potentially, we could look at increasing it to the same as we have as the upload speed in SmartSpin2k (921600), providing it tests okay.